### ISSUE TITLE
publish tree-sitter-rst as a python package. 

### ISSUE THREAD

**Issue by Carreau** [jupyter/papyri#171]
This will let user not build the tree-sitter-rst package.

**Comment by Carreau** [comment]
This should now be part of https://github.com/grantjenks/py-tree-sitter-languages, except i can't get it to work on mac...

**Comment by melissawm** [comment]
In the meantime, could this be a git submodule?

**Comment by Carreau** [comment]
Yes, that is a good idea. We could also re-work with upstream to see if they can publish wheels as well.
